infra_tooling_account/infra_tooling_account/grafana/cloudwatch_dashboard.template.json

```diff
@@ -390,7 +418,60 @@
             "properties": [
               {
                 "id": "custom.width",
-                "value": 133
+                "value": 114
+              },
+              {
+                "id": "links"
+              },
+              {
+                "id": "displayName",
+                "value": "Event Result"
+              }
+            ]
+          },
+          {
+            "matcher": {
+              "id": "byName",
+              "options": "execution_info_url"
+            },
+            "properties": [
+              {
+                "id": "links",
+                "value": [
+                  {
+                    "targetBlank": true,
+                    "title": "Link",
+                    "url": "${__value.raw}"
+                  }
+                ]
+              },
+              {
+                "id": "displayName",
+                "value": "Execution URL"
```

Comment:
for display name, maybe, we can think of something like "Link to Execution in AWS Console".
Just wondering if it's a bit more user-friendly. Your ideas on naming here are appreciated

Reply:
sure, thank you for your comments :)

Reply:
I've decided to name it a bit shorter as "Execution URL" , also I've updated the link text (so to simply display "Link" instead of long link) 

![image](https://github.com/Soname-Solutions/salmon/assets/150046345/7812306d-60f4-443b-90fe-543b3336dd2b)

Reply:
renamed as discussed in the meeting